src/main/sensors/gyro.c

```diff
@@ -183,15 +183,15 @@ PG_RESET_TEMPLATE(gyroConfig_t, gyroConfig,
     .gyro_hardware_lpf = GYRO_HARDWARE_LPF_NORMAL,
     .gyro_32khz_hardware_lpf = GYRO_32KHZ_HARDWARE_LPF_NORMAL,
     .gyro_lowpass_type = FILTER_PT1,
-    .gyro_lowpass_hz = 90,
+    .gyro_lowpass_hz = 100,
     .gyro_lowpass2_type = FILTER_PT1,
-    .gyro_lowpass2_hz = 0,
+    .gyro_lowpass2_hz = 300,
     .gyro_high_fsr = false,
     .gyro_use_32khz = false,
     .gyro_to_use = GYRO_CONFIG_USE_GYRO_DEFAULT,
-    .gyro_soft_notch_hz_1 = 400,
+    .gyro_soft_notch_hz_1 = 0,
     .gyro_soft_notch_cutoff_1 = 300,
-    .gyro_soft_notch_hz_2 = 200,
+    .gyro_soft_notch_hz_2 = 0,
     .gyro_soft_notch_cutoff_2 = 100,
```

Comment:
just to have it clean we may also should set both cutoffs to 0.
this also is needed to for the configurator. since it checkes for both values to be 0 to show it as disabled

Reply:
The configurator only looks at the hz, not the cutoff:

https://github.com/betaflight/betaflight-configurator/blob/5f68d5982c94c6c2266db11420e557df0f8a9ccb/src/js/tabs/pid_tuning.js#L286

But yes, is better to let both as zero if we don't have any "default" value.